src/index.js

```diff
@@ -12,10 +12,15 @@ const { version, description } = require('../package.json')
 const {
   mutations: {
     createEtchPacket: {
-      getMutation: getCreateEtchPacketMutation,
+      generateMutation: generateCreateEtchPacketMutation,
     },
     generateEtchSignUrl: {
-      getMutation: getGenerateEtchSignUrlMutation,
+      generateMutation: generateGenerateEtchSignUrlMutation,
```

Comment:
Ya double `generated` here...

```suggestion
      generateMutation: generateEtchSignUrlMutation,
```

Reply:
Ta da fixed! Also added providerConfig to getEtchPacket's defaultResponseQuery. Need that to access organizationSlug to be used in the example repo.